downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -228,9 +228,17 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 11 (2201.11.0)
   export = true
   readme = "README.md"
    ```
-  
+
 - Upgraded the version of BALA archives to `3.0.0` to accommodate the above changes.
 
 ### Bug fixes
 
 ## Backward-incompatible changes
+
+## External Contributors
+
+We would like to extend our heartfelt thanks to the following external contributors for their valuable contributions to
+the Ballerina SwanLake Update 11 release.
+
+* [Randil Tharusha](https://github.com/randilt)
+* [Thilan Dissanayaka](https://github.com/thil4n)
```

Comment:
```suggestion
* [Randil Tharusha](https://github.com/randilt)
* [Thilan Dissanayaka](https://github.com/thil4n)
* [DaAlbrecht](https://github.com/DaAlbrecht)
```
DaAlbrecht added secure socket support to ldap: https://github.com/ballerina-platform/module-ballerina-ldap/pull/66